src/ui/views/ui_app.jade

```diff
@@ -125,7 +125,7 @@ block content
           err.style['text-align'] = 'center';
           err.style['background'] = '#F44336';
           err.style['padding'] = '25px';
-          err.innerText = '#{i18n('UI-WELCOME_ERROR')}';
+          err.innerText = '#{i18n_singleQuoteVar('UI-WELCOME_ERROR')}';
 
```

Comment:
what about something like `err.innerText = "#{i18n('UI-WELCOME_ERROR').replace(/"/g,'\\"')}";` for this one spot (if we don't use the meta tag option)

_you are in a maze of twisty nested quoted strings, some alike_